modules/spo-recording-profiles.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-recording-profiles_{context}"]
+= Recording profiles from workloads
+
+The Security Profiles Operator can record {type} profiles with `ProfileRecording` objects.
+
+When using the log enricher for recording {type} profiles, ensure that the log enricher feature is enabled. See _Additional resources_ for more information.
+
+[NOTE]
+====
+A container with `privileged: true` security context restraints prevents log-based recording. Privileged containers are not subject to {type} policies and log-based recording makes use of a special {type} profile to record events.
+====
+
+.Procedure
+
+. Create a `ProfileRecording` containing a `recorder: logs` variable:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
+kind: ProfileRecording
+metadata:
+  name: test-recording
+spec:
+  kind: {kind}
+  recorder: logs
+  podSelector:
+    matchLabels:
+      app: my-app
+----
+
+. Create a workload to record:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: my-pod
+  labels:
+    app: my-app
+spec:
+  containers:
+    - name: nginx
+      image: quay.io/security-profiles-operator/test-nginx:1.19.1
+    - name: redis
+      image: quay.io/security-profiles-operator/redis:6.2.1
+----
+
+. Confirm the pod is in a `Running` state by entering the following command:
++
+[source,terminal]
+----
+$ oc get pods
+----
++
+.Example output
+[source,terminal]
+----
+NAME     READY   STATUS    RESTARTS   AGE
+my-pod   2/2     Running   0          18s
+----
+
+. The enricher indicates that it receives audit logs for those containers:
++
+[source,terminal]
+----
+$ oc -n security-profiles-operator logs --since=1m --selector name=spod log-enricher
+----
++
+.Example output
+[source,terminal]
+----
+…
+I0705 12:08:18.729660 1843190 enricher.go:136] log-enricher "msg"="audit"  "container"="redis" "executable"="/usr/local/bin/redis-server" "namespace"="default" "node"="127.0.0.1" "pid"=1847839 "pod"="my-pod" "syscallID"=232 "syscallName"="epoll_wait" "timestamp"="1625486870.273:187492" "type"="{type}"
+----
+
+.Verification
+
+. Remove the pod:
++
+[source,terminal]
+----
+$ oc delete pod my-pod
```

Comment:
Could add the `-n security-profiles-operator` flag here to be consistent with other examples.